week1/index.js

```diff
@@ -1,38 +1,30 @@
-import HTTP from 'http'
-import Path from 'path'
-
-import sendIndexHTML from './responses/sendIndexHTML'
-import sendPage2HTML from './responses/sendPage2HTML'
-import sendStylesCSS from './responses/sendStylesCSS'
-import sendText from './responses/sendText'
-
-const server = HTTP.createServer((request, response) => {
-	console.log(request.method, request.url)
-
-	switch (request.url) {
-	case '/':
-		sendIndexHTML(response)
-		break
-	case '/page2':
-		sendPage2HTML(response)
-		break
-	case '/styles.css':
-		sendStylesCSS(response)
-		break
-	default:
-		const extension = Path.extname(request.url)
-		if (extension === '') {
-			response.statusCode = 302
-			response.setHeader('Location', '/')
-		} else {
-			response.statusCode = 404
-			sendText(response, "File not found")
-		}
-	}
+const http = require('http');
+const port = 3000;
+let state = 10;
+let server = http.createServer((request, response)=>{
 	
-	response.end()
-})
-
-server.listen(3001)
+	
+	if (request.url === "/add"){
+		state +=1
+		response.write(`<h1>The state is : </h1><h2>${state}</h2>`)
 
-console.log('Server started')
\ No newline at end of file
+	}else if (request.url === "/remove"){
+		state -=1
+		response.write(`<h1>The state is : </h1><h2>${state}</h2>`)
+	
+	}else if (request.url === "/reset"){
+		state = 10
+		response.write(`<h1>The state is : </h1><h2>${state}</h2>`)
+		
+	}else if (request.url === "/state"){
+		response.write(`<h1>The state is :</h1><h2>${state}</h2>`)
+		
+	}else{
+		response.write(`<h1> Error 404 Invalid Request!</h1>`)
```

Comment:
Since you haven't set an http status code, this will probably return 200 (the default). In the case of a 'not found', we probably want to return with a status code of 404 - can you make this change? Check the Node HTTP documentation for details.

Also note that "Invalid Request" is usually associated with status 400, and is different from 404 "Not Found". Check for details: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes